### PR TITLE
Add trunk instrcutions

### PR DESCRIPTION
**Pull request by LeoDog896**
(My "c" key was broken at the time of writing)

This PR is meant to ease contributing by using nightly in the build folder itself and adding the trunk dependency requirement in CONTRIBUTING.md

### CODE REVIEW

File: rust-toolchain [hunk 1]
```diff
@@ -0,0 +1 @@
+nightly
```
Comment (svenstaro) [inline rust-toolchain:1]: Why depend on `nightly`? I'd prefer if we used `stable` for development.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -3,6 +3,10 @@
 If you want to add a cool module just implement it, lint it with [clippy](https://github.com/rust-lang-nursery/rust-clippy)
 and make a pull request with a screenshot. I will probably accept it.
 
+## Building for the web
+
+The `trunk` cli is required to build the web version. To add this, run `cargo install trunk`.
```
Comment (svenstaro) [inline CONTRIBUTING.md:8]: Good idea to document this.

**Review by svenstaro — CHANGES_REQUESTED** [review]

File: rust-toolchain (hunk 1 above)
Comment (LeoDog896) [reply rust-toolchain:1]: Dependencies didn't compile without it but I'll switch it to stable.

File: rust-toolchain [hunk 3]
```diff
@@ -1 +1 @@
-nightly
\ No newline at end of file
+stable
```
Comment (svenstaro) [inline rust-toolchain:1]: Coming back to this, I don't want to force a toolchain at all in this project. I don't think there's an advantage to this. It should realistically work on any toolchain.

Comment (LeoDog896) [reply rust-toolchain:1]: Got it -- I think the versions are installed weird on a fresh install (Gitpod / GitHub Workspaces) thus requiring nightly.

**Comment by svenstaro** [comment]
Thanks!

**Pull request merged**